### PR TITLE
Show context menu when selecting photo.

### PR DESCRIPTION
**Pull request by a2444mo10**
Implemented to show the context menu on iOS13 later.
The context menu allows you to select and deselect photos.
Tap the preview to transition to the details screen.

### CODE REVIEW

File: NohanaImagePicker/ImagePreviewViewController.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
+//
+//  ImagePreviewViewController.swift
+//  NohanaImagePicker
+//
+//  Created by atsushi.yoshimoto on 2021/12/08.
+//  Copyright © 2021 nohana. All rights reserved.
+//
```
Comment (tyokujin) [inline NohanaImagePicker/ImagePreviewViewController.swift:7]: Please modified copyright and license.

File: NohanaImagePicker/AssetListViewController.swift [hunk 2]
```diff
@@ -108,6 +108,51 @@ class AssetListViewController: UICollectionViewController, UICollectionViewDeleg
...
+                let previewViewController = ImagePreviewViewController(asset: asset)
+                let imageSize = cell.imageView.image?.size ?? .zero
+                let width = self.view.bounds.width
+                let height = imageSize.height * (width / imageSize.width)
+                let contentSize = CGSize(width: width, height: height)
+                previewViewController.preferredContentSize = contentSize
+                return previewViewController
+            }, actionProvider: { _ in
+                if nohanaImagePickerController.pickedAssetList.isPicked(asset) {
+                    let deselect = UIAction(title: "選択を外す", image: UIImage(systemName: "minus.circle"), attributes: [.destructive]) { _ in
```
Comment (tyokujin) [inline NohanaImagePicker/AssetListViewController.swift:128]: Please implement `NSLocalizedString`.
As described link.
https://github.com/nohana/NohanaImagePicker/blob/10e97c282ec21da2ef914a1ed9f9600f8229b640/NohanaImagePicker/AlbumListViewController.swift#L45

Comment (a2444mo10) [reply NohanaImagePicker/AssetListViewController.swift:128]: OK.

File: NohanaImagePicker/ImagePreviewViewController.swift [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    init(asset: PhotoKitAsset) {
+        super.init(nibName: nil, bundle: nil)
+
+        asset.image(targetSize: UIScreen.main.bounds.size) { [weak self] (imageData) -> Void in
+            guard let self = self else { return }
+            DispatchQueue.main.async {
+                if let imageData = imageData {
+                    self.imageView.image = imageData.image
+                }
+            }
```
Comment (tyokujin) [inline NohanaImagePicker/ImagePreviewViewController.swift:37]: I think better to write `[weak self]` in `Dispatch Queue`

**Review by tyokujin — COMMENTED** [review]
Thanks your coding! Please check my comments.

**Comment by a2444mo10** [comment]
@tyokujin 
Fixed the code.
Please review again.

**Review by tyokujin — APPROVED** [review]

**Pull request merged**